src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -135,7 +135,7 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
             pnts: this.pntsParse.bind(this),
         };
 
-        return Fetcher.arrayBuffer(url).then((result) => {
+        return Fetcher.arrayBuffer(url, layer.fetchOptions).then((result) => {
```

Comment:
There's a different set of options for `Fetcher.texture` (which uses TextureLoader under the hood) and `Fetcher.{json,xml}`. They don't overlap, but I thought it was clearer that way. Now I don't really know, your call :-)